tsconfig.json

```diff
@@ -18,6 +18,9 @@
     ],
     "resolveJsonModule": true,
     "noEmit": true,
-    "target": "esnext"
+    "target": "esnext",
+    "paths": {
+      "@openmrs/*": ["./node_modules/@openmrs/*"]
```

Comment:
Unrelated; this rule is in all our other tsconfigs and prevents an annoying error.